apps/prairielearn/src/pages/administratorQueries/administratorQueries.html.ts

```diff
@@ -0,0 +1,56 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+interface Query {
+  description: string;
+  params: string[];
+  sqlFilename: string;
+  link: string;
+}
+
+export function AdministratorQueries({
+  queries,
+  resLocals,
+}: {
+  queries: Query[];
+  resLocals: Record<string, any>;
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", resLocals)}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar') %>", {
+          ...resLocals,
+          navPage: 'admin',
+          navSubPage: 'queries',
+        })}
+        <main id="content" class="container-fluid">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Queries</div>
+            <div class="table-responsive">
+              <table class="table table-sm table-hover table-striped">
+                <tbody>
+                  ${queries.map(
+                    (query) => html`
+                      <tr>
+                        <td>
+                          <a href=${`../administrator/query/${query.link}`}>
+                            <code>${query.sqlFilename}</code>
```

Comment:
The <tt> tag is deprecated (per MDN docs, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt). Replaced with a <code> tag.